test_gnupg.py

```diff
@@ -545,13 +546,14 @@ def test_list_subkey_after_generation(self):
 
         self.assertEqual(key_info['subkeys'][1][1], 'e')
         self.assertEqual(key_info['subkeys'][1][2], subkey_encrypt.fingerprint)
-        for skid, _, sfp in key_info['subkeys']:
+        for skid, _, sfp, grp in key_info['subkeys']:
             self.assertTrue(skid in skinfo)
             info = skinfo[skid]
             self.assertEqual(info['keyid'], skid)
             self.assertEqual(info['type'], 'sub')
             self.assertTrue(sfp in public_keys.key_map)
             self.assertTrue(public_keys.key_map[sfp] is key_info)
+            self.assertIsNotNone(grp)
```

Comment:
Could this check be stronger? For example, `self.assertTrue(grp)` would check for a false value in general, such as an empty string. Same in other places where `assertIsNotNone` is used.